pvacseq/lib/vaccine_design.py

```diff
@@ -0,0 +1,378 @@
+#  Input would be a protein fasta
+# ie:
+# python vaccine_design.py test peptides.fa -o output.csv ann  H-2-Kb -e 8
+import shutil
+import sys
+from pathlib import Path
+root = str(Path(__file__).resolve().parents[1])
+sys.path.append(root)
+
+try:
+    from .. import lib
+except ValueError:
+    import lib
+
+import pandas
+import networkx as nx
+import itertools
+from Bio import SeqIO
+import lib
+from parse_output import *
```

Comment:
I don't think you ever use any of the functions from this file.  This is also causing issues since `parse_output.py` has been removed on the `develop` branch